third_party/move/move-model/bytecode/src/stackless_control_flow_graph.rs

```diff
@@ -32,6 +32,16 @@ pub enum BlockContent {
     Dummy,
 }
 
+impl BlockContent {
+    pub fn to_bytecodes<'a>(&self, bytecodes: &'a [Bytecode]) -> &'a [Bytecode] {
```

Comment:
Where is this used?  I can't find it with a search in github.

Reply:
This is one use in `ControlFlowGraphCodeGenerator::new`